code/game/machinery/iv_drip.dm

```diff
@@ -1,101 +1,128 @@
 /obj/machinery/iv_drip
 	name = "\improper IV drip"
 	icon = 'icons/obj/iv_drip.dmi'
+	icon_state = "iv_drip"
 	anchored = 0
 	density = 1
+	var/mob/living/carbon/human/attached = null
+	var/mode = 1 // 1 is injecting, 0 is taking blood.

```

Comment:
Rename this variable to something more obvious like `injecting`, *or* use defines.